libs/application/core/src/validation/validators.ts

```diff
@@ -49,37 +49,44 @@ function partialSchemaValidation(
     try {
       trimmedSchema.parse({ [key]: answer })
     } catch (e) {
-      error = populateError(error, e, newPath)
-
+      const zodErrors: ZodSuberror[] = e.errors
+      const keyIsIncludedInErrors = zodErrors.some((err) =>
+        err.path.includes(key),
```

Comment:
Is this a good enough check, eg if there are two `name` fields in different places? Can we check that the path starts with `constructedErrorPath`?

Reply:
Good question, this code was actually some leftover since debugging and is not needed at all 😄